crates/flipperzero/examples/dialog.rs

```diff
@@ -11,8 +11,10 @@ extern crate flipperzero_alloc;
 
 use core::ffi::CStr;
 
-use flipperzero::gui::canvas::Align;
-use flipperzero::dialogs::{self, DialogMessage, DialogMessageButton, DialogsApp};
+use flipperzero::{
```

Comment:
Was this change intended?

Reply:
Yes. There was some update of `use` order from Clippy, and then I applied nesting.